scripts/dts/python-devicetree/tests/test-bindings/enums.yaml

```diff
@@ -32,5 +32,11 @@ properties:
       - whitespace is ok
       - 123 is ok
 
+  array-enum:                   # tokenizable string-array
+    type: string-array
```

Comment:
please also add a test case for the `array` type

Reply:
done